### PR TITLE
Update journalist_user_manual.md

### PR DESCRIPTION
**Pull request by mr-grooms**
Misspelling.

### CODE REVIEW

**Comment by conorsch** [comment]
@mr-grooms The spelling is inconsistent within the `journalist_user_manual.md` document, and so should indeed be updated, but it *is* consistent with the spelling from the [MAT: Metadata Anonymisation Toolkit website](https://mat.boum.org/). Therefore I think we should standardize on the "Anonymisation" spelling, to match the project's documentation.

By the way, we're working on a substantial docs overhaul, so now's a great time to have this discussion!

**Comment by garrettr** [comment]
Agree that we should standardize on "Anonymisation" to be consistent with the upstream project's website and documentation.

**Comment by conorsch** [comment]
Thanks, @garrettr. @mr-grooms, if you're able to make the change from "Anonymization" to "Anonymisation" throughout—only two uses of the term, but my count—I'd be happy to merge.

File: docs/journalist_user_manual.md [hunk 1]
```diff
@@ -125,4 +125,4 @@ When you are done you will have another document with the same filename but endi
 
 Plug the `Transfer Device` into your workstation computer and copy the encrypted documents to it. Decrypt them with `gnupg`.
 
-Write articles and blog posts, edit video and audio, and publish. Expose crimes and corruption, and change the world.
\ No newline at end of file
+Write articles and blog posts, edit video and audio, and publish. Expose crimes and corruption, and change the world.
```
Comment (garrettr) [inline docs/journalist_user_manual.md]: I can't see what changed here. Trailing newline?

**Pull request closed without merging**